example/zrc7/zrc7-metadata.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
The schema from eip-1155 is represented in JSON and it is not a template. You misunderstood the JSON Metadata schema. 

Remove this file. We already have the example in the ZRC-7 markdown.